content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fariness,
```

Comment:
nit:
```suggestion
Each apiserver independently applies prioritization and fairness,
```

Alternatively:
```suggestion
If the `RequestManagement` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/] is enabled, the API server prioritizes incoming requests, based on the
configuration you have set.
```
(fine to wrap over several lines, of course)
```

Reply:
Depending on other changes, here might be a good point to introduce the resource names:
`PriorityLevelConfiguration` and `FlowSchema`.

Reply:
I thought about that, and deliberately draw a distinction between (a) the `PriorityLevelConfiguration` Kind of API object and (b) the concept of a _priority level_.  A priority level has a set of queues at each apiserver; a PriorityLevelConfiguration is just an API object that configures a priority level and does not have queues.

Reply:
How I'd help the reader here: have a fragment later that describes those API objects, and make “API objects” hyperlink to that fragment.